.github/actions/install-dashboards/action.yml

```diff
@@ -68,14 +80,30 @@ runs:
         node-version: ${{ steps.tool-versions.outputs.node_version }}
         registry-url: 'https://registry.npmjs.org'
 
-    - run: |
+    - name: Setup Opensearch Dashboards on Linux
+      if: ${{ runner.os == 'Linux' }}
+      run: |
         npm uninstall -g yarn
         echo "Installing yarn ${{ steps.tool-versions.outputs.yarn_version }}"
         npm i -g yarn@${{ steps.tool-versions.outputs.yarn_version }}
+        yarn cache clean
+        yarn add sha.js
       working-directory: OpenSearch-Dashboards
       shell: bash
 
-    - uses: nick-fields/retry@v2
+    - name: Setup Opensearch Dashboards on Windows
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        npm uninstall -g yarn
+        echo "Installing yarn ${{ steps.tool-versions.outputs.yarn_version }}"
+        npm i -g yarn@${{ steps.tool-versions.outputs.yarn_version }}
+        yarn cache clean
+        yarn add sha.js
+      working-directory: OpenSearch-Dashboards
+      shell: pwsh
+
+    - name: Bootstrap the OpenSearch Dashboard
```

Comment:
This name might be incorrect as this step retries any failed github steps. You can remove this name or rename it to `Retry failed steps`

Reply:
Thanks for the comment DC! I think this is the step of Bootstrap, but we were just using this action for another retry attempt. If you see into the `with: ` you can see the actual command it runs with retry, which is `command: yarn --cwd OpenSearch-Dashboards osd bootstrap --oss`